### PR TITLE
Move quantization format from TPC to exporter API and add MCTQ option

### PR DESCRIPTION
**Pull request by reuvenperetz**
## Pull Request Description:

Add MCTQ to quantization format (in addition to fake-quant and int8).
Move quantization format from TPC to exporter API.
Change default pytorch export from torchscript-fake-quant to onnx-mctq.
Change default keras export to h5 (or .keras) mctq.

Remove some old exporter tests that are duplicated.

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/exporter/model_exporter/keras/mctq_keras_exporter.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            save_model_path: Path to save the exported model.
+        """
+
+        super().__init__(model,
+                         is_layer_exportable_fn,
+                         save_model_path)
+
+    def export(self):
+        """
+        Convert an exportable (fully-quantized) Keras model to a fakely-quant model
```
Comment (Idan-BenAmi) [inline model_compression_toolkit/exporter/model_exporter/keras/mctq_keras_exporter.py:43]: update description: ...Save Keras model with custom MCT quantizer layers...

File: model_compression_toolkit/exporter/model_exporter/pytorch/pytorch_export_facade.py [hunk 2]
```diff
@@ -32,17 +31,15 @@
...
-                             target_platform_capabilities: TargetPlatformCapabilities,
                              is_layer_exportable_fn: Callable = is_pytorch_layer_exportable,
-                             serialization_format: PytorchExportSerializationFormat =
-                             PytorchExportSerializationFormat.TORCHSCRIPT,
-                             use_onnx_custom_quantizer_ops: bool = False) -> None:
+                             serialization_format: PytorchExportSerializationFormat = PytorchExportSerializationFormat.ONNX,
+                             quantization_format : QuantizationFormat = QuantizationFormat.MCTQ) -> None:
         """
         Export a PyTorch quantized model to a torchscript or onnx model.
         The model will be saved to the path in save_model_path.
```
Comment (Idan-BenAmi) [inline model_compression_toolkit/exporter/model_exporter/pytorch/pytorch_export_facade.py:45]: description - include the new quantizationFormat

**Review by Idan-BenAmi — APPROVED** [review]
